src/main/java/io/aiven/kafka/connect/gcs/GcsSinkTask.java

```diff
@@ -72,6 +78,7 @@ public void start(final Map<String, String> props) {
         this.storage = StorageOptions.newBuilder()
                 .setHost(config.getGcsEndpoint())
                 .setCredentials(config.getCredentials())
+                .setHeaderProvider(FixedHeaderProvider.create(USER_AGENT_HEADER_KEY, USER_AGENT_HEADER_VALUE))
```

Comment:
Do we need to have a way to provide a different value?

Reply:
wdym?